clang/unittests/ASTMatchers/Dynamic/VariantValueTest.cpp

```diff
@@ -1,10 +1,10 @@
-//===- unittest/ASTMatchers/Dynamic/VariantValueTest.cpp - VariantValue unit tests -===//
+// unittest/ASTMatchers/Dynamic/VariantValueTest.cpp - VariantValue unit tests
```

Comment:
Repeating the full path name seems unnecessary? (In various files.)

Reply:
Fixed.